sdk/providerhub/Microsoft.Azure.Management.ProviderHub/tests/Microsoft.Azure.Management.ProviderHub.Tests.csproj

```diff
@@ -0,0 +1,30 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$(RepoEngPath)/mgmt/AzSdk.test.reference.props" />
+  <PropertyGroup>
+    <PackageId>ProviderHub.Tests</PackageId>
+    <Description>ProviderHub.Tests Class Library</Description>
+    <AssemblyName>ProviderHub.Tests</AssemblyName>
+    <Version>1.0.0</Version>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.Management.Storage" Version="4.2.0-preview" />
+    <PackageReference Include="Microsoft.Azure.Management.Network" Version="10.0.0-preview" />
+    <PackageReference Include="Microsoft.Azure.Management.ResourceManager" Version="1.6.0-preview" />
+    <PackageReference Include="Microsoft.Azure.Graph.RBAC" Version="2.2.2-preview" />
+    <PackageReference Include="WindowsAzure.Storage" Version="8.7.0" />
+    <ProjectReference Include="$(AzureCoreTestFramework)" />
+    <ProjectReference Include="..\src\Microsoft.Azure.Management.ProviderHub.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <None Update="SessionRecords\**\*.json">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+  </ItemGroup>
+
+
+  <ItemGroup>
```

Comment:
don't need this

Reply:
The lines highlighted don't seem to have any service reference, could you please clarify which one?